Swagger/PlayFab/Admin.swagger.json

```diff
@@ -4116,8 +4116,7 @@
       "properties": {
         "code": {
           "description": "Numerical HTTP code",
-          "type": "number",
-          "x-actualtype": "int32"
```

Comment:
Please don't remove x-actualType.
Go ahead and change type, but don't remove x-fields we need.